db/re/item_synthesis.yml

```diff
@@ -0,0 +1,2447 @@
+  SourceNeeded: 2
+  SourceItem:
+  - Item: 25729
+    Amount: 30
+  - Item: 7450
+    Amount: 50
+  Reward: getgroupitem(IG_Bs_Sha_M_S_3);
+- Id: 23889 # Bs_Sha_M_S_4
+  SourceNeeded: 2
+  SourceItem:
+  - Item: 25729
+    Amount: 30
+  - Item: 25629
+    Amount: 50
+  Reward: getgroupitem(IG_Bs_Sha_M_S_4);
+- Id: 23890 # Bs_Sha_M_S_12
+  SourceNeeded: 2
+  SourceItem:
+  - Item: 25729
+    Amount: 30
+  - Item: 7169
+    Amount: 50
+  Reward: getgroupitem(IG_Bs_Sha_M_S_12);
+- Id: 23891 # Bs_Sha_M_S_14
+  SourceNeeded: 2
+  SourceItem:
+  - Item: 25729
+    Amount: 30
+  - Item: 7019
+    Amount: 50
+  Reward: getgroupitem(IG_Bs_Sha_M_S_14);
+- Id: 23892 # Bs_Sha_M_S_24
+  SourceNeeded: 2
+  SourceItem:
+  - Item: 25729
+    Amount: 25
+  - Item: 7513
+    Amount: 150
+  Reward: getgroupitem(IG_Bs_Sha_M_S_24);
+- Id: 23893 # Bs_Sha_M_S_25
+  SourceNeeded: 2
+  SourceItem:
+  - Item: 25729
+    Amount: 25
+  - Item: 7300
+    Amount: 150
+  Reward: getgroupitem(IG_Bs_Sha_M_S_25);
+- Id: 23665 # PhysicalMagical_Mix
+  SourceNeeded: 3
+  NeedRefineMin: 7
+  SourceItem:
+  - Item: 24019
+  - Item: 24018
+  - Item: 24020
+  - Item: 24022
+  - Item: 24021
+  - Item: 24023
+  Reward: getgroupitem(IG_PhysicalMagical_Mix);
+- Id: 23666 # ImmunedAthena_Mix
+  SourceNeeded: 2
+  NeedRefineMin: 7
+  SourceItem:
+  - Item: 24027
+  - Item: 24026
+  - Item: 24051
+  Reward: getitem(24399,1); // 이뮨드 아테나 쉴드 쉐도우
+- Id: 23667 # HardChamption_Mix
+  SourceNeeded: 2
+  NeedRefineMin: 7
+  SourceItem:
+  - Item: 24028
+  - Item: 24049
+  - Item: 24025
+  Reward: getitem(24400,1); // 하드 챔피언 슈즈 쉐도우
+- Id: 23668 # KingbirdAncient_Mix
+  SourceNeeded: 2
+  NeedRefineMin: 7
+  SourceItem:
+  - Item: 24031
+  - Item: 24029
+  Reward: getitem(24401,1); // 킹버드 에이션트 아머 쉐도우
+- Id: 23669 # CriticalHit_Mix
+  SourceNeeded: 2
+  NeedRefineMin: 7
+  SourceItem:
+  - Item: 24333
+  - Item: 24332
+  - Item: 24032
+  - Item: 24030
+  Reward: getitem(15280,1); // 크리티컬 히트 아머 쉐도우
+- Id: 23545 # Silver_Statue
+  SourceItem:
+  - Item: 28942
+  Reward: getgroupitem(IG_Silver_Statue);
+- Id: 23474 # InfinityShadow_Mix
+  SourceNeeded: 2
+  NeedRefineMin: 7
+  SourceItem:
+  - Item: 24150
+  - Item: 24151
+  Reward: getitem(24386,1); // 인피니티 웨폰 쉐도우
+- Id: 23324 # Stability Shadow Box // 스테빌리티S변환상자
+  SourceNeeded: 3
+  NeedRefineMin: 5
+  SourceItem:
+  - Item: 24091
+  - Item: 24092
+  - Item: 24093
+  - Item: 24094
+  - Item: 24095
+  - Item: 24096
+  - Item: 24097
+  - Item: 24098
+  - Item: 24099
+  - Item: 24100
+  - Item: 24101
+  - Item: 24102
+  - Item: 24103
+  - Item: 24104
+  - Item: 24105
+  - Item: 24106
+  - Item: 24107
+  - Item: 24108
+  Reward: getgroupitem(IG_Stability_Shadow_Box);
+- Id: 23308 # 부스터증폭기
+  SourceItem:
+  - Item: 19241
+  Reward: |
+    if (rand(2) == 0) {
+        getitem(19245,1); // chance to become Crimson Booster
+    } else {
+        // Magical Booster Enchant (19241)
+        callfunc("F_Lapine_Magical_Booster_Enchant", .@opts[0], .@vals[0]);
```

Comment:
Change 
`callfunc("F_Lapine_Magical_Booster_Enchant", .@opts[0], .@vals[0]);`
to
`callfunc("F_Lapine_Magical_Booster_Enchant", .@opts[0], .@vals[0], .@opts[1], .@vals[1]);`